playground/frontend/lib/modules/examples/repositories/models/get_example_request.dart

```diff
@@ -23,4 +23,14 @@ class GetExampleRequestWrapper {
   final SDK sdk;
 
   GetExampleRequestWrapper(this.path, this.sdk);
+
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      other is GetExampleRequestWrapper &&
+          path == other.path &&
+          sdk == other.sdk;
+
+  @override
+  int get hashCode => path.hashCode ^ sdk.hashCode;
```

Comment:
Nit: Flutter has the [hashValues](https://api.flutter.dev/flutter/dart-ui/hashValues.html) function which is more standard.

Reply:
Hello, @KonradJanica! Thank you for the review, fixed that in the last commit.